docs/templates.rst

```diff
@@ -1140,11 +1140,23 @@ for Python objects such as strings and numbers.  The following literals exist:
     arguments to function calls and filters, or just to extend or include a
     template).
 
-42 / 42.23:
-    Integers and floating point numbers are created by just writing the
-    number down.  If a dot is present, the number is a float, otherwise an
-    integer.  Keep in mind that, in Python, ``42`` and ``42.0``
+42 / 123_456_789:
+    Integer numbers are created by just writing the number down. 
+    If a dot is present, the number will be considered a float.  
+    Keep in mind that, in Python, ``42`` and ``42.0``
     are different (``int`` and ``float``, respectively).
+    When dealing with long integers, it is possible to use the '_' character
+    to group the digits for legibility, eg: 100_000_000_000.
```

Comment:
\`\`100_000_000_000\`\`?

Reply:
Yes, those are now valid in Python. 
https://www.python.org/dev/peps/pep-0515/